hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -161,61 +168,62 @@ This is intended to be unique but also provide structured information to the DHC
 Specify DHCP_OPTION97=0 to revert the the old behaviour or a non-zero hex-value to specify a custom 4-byte prefix.
 
 Default: 0x34695052  
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
 ### Static IP address configuration
 If TFTP_IP and the following options are set then DHCP is skipped and the static IP configuration is applied. If the TFTP server is on the same subnet as the client then GATEWAY may be omitted.
 
 #### CLIENT_IP
 The IP address of the client e.g. "192.168.0.32"   
+
 Default: ""  
 Version: pieeprom-2020-04-16.bin  
 
 #### SUBNET
 The subnet address mask e.g. "255.255.255.0"   
+
 Default: ""  
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
 #### GATEWAY
-The gateway address to use if the TFTP server is on a differenet subnet e.g. "192.168.0.1"
+The gateway address to use if the TFTP server is on a differenet subnet e.g. "192.168.0.1"  
+
 Default: ""  
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
 #### MAC_ADDRESS
 Overrides the Ethernet MAC address with the given value. e.g. dc:a6:32:01:36:c2  
+
 Default: ""   
-Version: pieeprom-2020-04-16.bin
+Version: 2020-04-16  
 
 ### DISABLE_HDMI
 The [HDMI boot diagnostics](./boot_diagnostics.md) display is disabled if DISABLE_HDMI=1. Other non-zero values are reserved for future use.
 
 Default: 0   
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
-`Version: pieeprom-2020-06-15.bin - STABLE`
-The `disable_splash` property is no longer checked because the HDMI diagnostics screen is started before config.txt is read.
+From version 2020-09-03 the `disable_splash` property in `config.txt` is no longer checked because the HDMI diagnostics screen is started before `config.txt` is read.
 
 ### ENABLE_SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set `ENABLE_SELF_UPDATE=1` and add `bootloader_update=1` in config.txt.
-N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
-
-If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
+Enables the bootloader to update itself from a network or USB-MSD boot.
```

Comment:
`USB-MSD`  should perhaps be replaced with `USB mass storage` throughout this file, for ease of understanding. Alternatively explain the `MSD` initialism the first time it is used on the page.

Reply:
I've added it the first time but MSD is a common acronym and makes this less wordy